docs/data-formats.md

```diff
@@ -204,6 +204,16 @@ The `initialize` attribute is optional and when not specified the code to set th
 </linker>
 ```
 
+### Remove embedded resources
+
+```xml
+<linker>
+  <assembly fullname="Assembly">
+    <resource name="Resource" action="remove" />
```

Comment:
I think this will be best used with the feature condition. Could you add it to this PR?